core/orchestrator.go

```diff
@@ -208,6 +208,15 @@ func (orch *orchestrator) SufficientBalance(manifestID ManifestID) bool {
 	return true
 }
 
+func (orch *orchestrator) DebitFees(manifestID ManifestID, price *net.PriceInfo, pixels int64) {
+	// Don't debit in offchain mode
+	if orch.node == nil || orch.node.Recipient == nil || orch.node.Balances == nil {
+		return
+	}
+	amount := new(big.Rat).Mul(big.NewRat(price.GetPricePerUnit(), price.GetPixelsPerUnit()), big.NewRat(pixels, 1))
```

Comment:
I'm wondering under what conditions this would be possible (once we actually populate PriceInfo in OrchestratorInfo 😄 ) , considering we are enforcing the checks on O's BasePrice to not be those values when starting the node? 

It would require the `txCostOverhead` to be 0, which is impossible as well since that is calculated as `1+ 1/txCostMultiplier`  ? 
`new(big.Rat).Mul(orchBasePrice, txCostOverhead)`

And I don't think `txCostMultiplier` can be 0 either to cause a panic calculating the overhead unless the `gasPrice` is 0 AND the `ticketEV` is set to 0 ?

Reply:
> I'm wondering under what conditions this would be possible

Ah yeah I think we can skip the guard clause here if we implement checks for `payment.GetExpectedPrice()` in #955 which will likely involve rejection of the payment if the expected price is invalid. Will be good to keep note of this while implementing that task though